repository_cli/cli/records.py

```diff
@@ -262,8 +266,9 @@ def add_identifier(identifier: map, pid: str):
         invenio repository rdmrecords identifiers add -p "fcze8-4vx33"
         -i '{ "identifier": "10.48436/fcze8-4vx33", "scheme": "doi"}'
     """
-    identifier = json.loads(identifier)
-    if type(identifier) is not dict:
+    try:
+        identifier = json.loads(identifier)
```

Comment:
Again, type changes, that should be rewritten that the type of the variable is not changed